lib/components/command_bar.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:rtchat/models/commands.dart';
+import 'package:rtchat/models/channels.dart';
+import 'package:rtchat/models/adapters/actions.dart';
+
+class CommandBarWidget extends StatelessWidget {
+  // final void Function() onPressed;
+  final CommandsModel commandsModel;
+  final ChannelsModel channelsModel;
+  final FocusNode chatInputFocusNode;
```

Comment:
It's slightly more preferable to pass a callback instead of the entire focus node. This typically makes things easier to test.

```
final void Function() onUnfocus;
```

Then in the caller `CommandBarWidget(onUnfocus: () => chatInputFocusNode.unfocus());`